src/components/molecules/PageFilter/page-filter.md

```diff
@@ -84,6 +84,23 @@ interface PeriodFilterProps {
 
 Passing the `interface`/`type` to the `PageFilter` component isn't strictly necessary, but it does give you code completion, when defining which property you want to have parsed.
 
+### Debounce fields
+
+In some cases you will have a text field, which will update the url on every change unless you have a submit button. But without the submit button, you can use the `DebouncedInput` component to debounce the changes in an input and thus only update the url after a specific time.
+
+For example:
+
+```tsx
+import { Input } from "antd";
+import DebouncedInput from "@app/components/atoms/DebouncedInput/DebouncedInput";
+
+<PageFilter>
+  <FilterItem label="Search for name" name="search">
+    <DebouncedInput wait={500} renderInput={props => <Input {...props} />} />
```

Comment:
Could one argue that we could make it simpler by simply importing the Ant Design input in the `DebouncedInput` component instead? We could then make the `DebouncedInput` accept the same `Input` props.  Just a thought.

Reply:
It is a good point, and initially I did it like that, but then I made it a bit more generic, so any ant d input in theory can be used, like Select, InputNumber, and so on, but the question is of course if we would ever be in a situation where we would debounce anything else than an input 🤔